flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/table/KafkaDynamicTableFactoryTest.java

```diff
@@ -400,6 +407,49 @@ public void testTableSourceSetOffsetResetWithException() {
                                 ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, errorStrategy));
     }
 
+    @Test
+    public void testTableSourceWithRecordEvaluator() {
+        Map<String, String> tableOptions = getBasicSourceOptions();
+        tableOptions.put(
+                "scan.record.evaluator.class",
+                "org.apache.flink.streaming.connectors.kafka.testutils.MockRecordEvaluator");
```

Comment:
Would it be better to pass the value as `MockRecordEvaluator.class.getName()` similar to what `testPerPartitionWatermarkKafka` does to get TestPartitioner's class name?